artifacts/centos/centos.go

```diff
@@ -126,6 +128,16 @@ func getAdditionalTags(version, variant, candidate string) (additionalTags []str
 	return
 }
 
+func getImageArchitecture(arch string) string {
+	if arch == "x86_64" {
```

Comment:
please create constants for x86_64, amd64, aarch64, arm64 and use them in other places too

Reply:
Okay.

Reply:
What would be the benefit of using constants, e.g. `artifactsCommon.AARCH64`? IMO it just adds another reference to the package and makes the line longer.

There is no practice in this project to have constants, e.g. @0xFelix added env variables such as `"u1.small"` and `"centos.7"` as strings and not constants.

Reply:
For now, I'll have constants only here:
```
package common

const (
	X86_64  = "x86_64"
	AMD64   = "amd64"
	AARCH64 = "aarch64"
	ARM64   = "arm64"
	NOARCH  = "unknown"
)

func GetImageArchitecture(arch string) string {
	switch arch {
	case X86_64:
		return AMD64
	case AARCH64:
		return ARM64
	default:
		return NOARCH
	}
}
```